packages/vscode-extension/src/debugging/RadonCDPProxyDelegate.ts

```diff
@@ -18,7 +19,12 @@ export class RadonCDPProxyDelegate implements CDPProxyDelegate {
   public onDebuggerResumed = this.debuggerResumedEmitter.event;
   public onConsoleAPICalled = this.consoleAPICalledEmitter.event;
 
-  constructor(private sourceMapRegistry: SourceMapsRegistry) {}
+  constructor(
+    private sourceMapRegistry: SourceMapsRegistry,
+    patterns: string[]
```

Comment:
I'd keep "skipFiles" name here and in the class instance. The "skipFiles" concept have the semantics of regular and negated patterns that needs to be considered in particular order while "patterns" is more a generic term